resources/liquibase/001-initial-schema.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd" logicalFilePath="001-initial-schema.xml">
+  <changeSet id="001" author="guseyn">
+    <sql>
+      CREATE TABLE "user" (
+        login VARCHAR(64) PRIMARY KEY NOT NULL,
+        avatar_url VARCHAR(2000) NOT NULL,
+        created TIMESTAMPTZ DEFAULT now() NOT NULL
+      );
+      CREATE INDEX idx_user1 ON "user" (login);
+    </sql>
+    <sql>
+      CREATE TABLE project (
+        id SERIAL PRIMARY KEY,
+        submitter VARCHAR(64) NOT NULL REFERENCES "user"(login),
+        coordinates VARCHAR(64) NOT NULL,
+        platform VARCHAR(64) NOT NULL DEFAULT 'github',
+        created TIMESTAMPTZ DEFAULT now() NOT NULL
+      );
+      CREATE INDEX idx_project1 ON project (id, submitter, coordinates, platform);
+    </sql>
+    <sql>
+      CREATE TABLE review (
+        id SERIAL PRIMARY KEY,
+        project INT NOT NULL REFERENCES project(id),
+        author VARCHAR(64) NOT NULL REFERENCES "user"(login),
+        text TEXT NOT NULL,
+        created TIMESTAMPTZ DEFAULT now() NOT NULL
+      );
+      CREATE INDEX idx_review1 ON review (project, author, text);
+    </sql>
+    <sql>
+      <!-- value here can be eigher -1, 0 or 1 -->
+      CREATE TABLE review_vote (
```

Comment:
@Guseyn how about just `vote`?

Reply:
@yegor256 In the future we might have more vote tables, I would use specified name